packaging/macOS/pyinstaller.spec

```diff
@@ -2,71 +2,85 @@
 
 import os
 
- # Awesome hack to load `__version__`
+WORKSPACE_DIR = os.getenv("WORKSPACE_DIR")
+if WORKSPACE_DIR is None:
+    WORKSPACE_DIR = '../..'
+
+# Awesome hack to load `__version__`
 __version__ = None
-exec(open("../../parsec/_version.py", encoding="utf-8").read())
+exec(open(f"{WORKSPACE_DIR}/parsec/_version.py", encoding="utf-8").read())
```

Comment:
@Ironicbay do you know why we make reference of the version here while we don't need this in the `pyinstaller.spec` used to do the packaging on windows ? (cf. https://github.com/Scille/parsec-cloud/blob/master/packaging/win32/pyinstaller.spec)

It seems the windows pyinstaller.spec managed to completely avoid needing the path of the sources...

Reply:
iirc it's for the app's metadata accessible with a right click from the user (in a "more info" type menu), along with the app name, developer ID, etc. Seemed nice to have